### ISSUE TITLE
ERROR: Hyper-V is not supported on your OS => Ensure your are running Windows 10 Pro or higher

### ISSUE THREAD

**Issue by cifcloud** [boxfuse/boxfuse-issues#257]
I downloaded the latest boxfuse client update using boxfuse -u but I get the following error while using boxfuse rm to clean up after the build.

This is on a linux EC2 instance (Ubuntu 16.04.6 LTS). The rest of the build worked OK but this happened while cleaning up resources afterwards.

Any suggestions?

```
jenkins@ip-172-31-16-17:~$ boxfuse rm cifcloud/cifcloud-web:2.4.145
Boxfuse client v.1.33.4.1504
Copyright 2020 Axel Fontaine Labs GmbH. All rights reserved.

Account: cifcloud (cifcloud)

ERROR: Hyper-V is not supported on your OS => Ensure your are running Windows 10 Pro or higher
```

**Comment by axelfontaine** [comment]
Thanks for reporting this! I'll keep you posted on a fix. It will probably go out as part of our next update early next week.

**Comment by knocknarea** [comment]
@axelfontaine This is still there on v.1.35.1.1524 
automated builds are flagged as fails as a result (although the deployment took place)

**Comment by axelfontaine** [comment]
This should now be fixed with the new `1.35.2.1525` client.

**Comment by cifcloud** [comment]
@axelfontaine Works well. Thanks!
